invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentItems/DocumentItems.js

```diff
@@ -53,16 +53,16 @@ export default class DocumentItems extends Component {
     });
   }
 
-  _render_table(data) {
+  render_table(data) {
```

Comment:
To keep the same style:
```suggestion
  renderTable(data) {
```